src/Microsoft.AspNetCore.All/runtime.json.in

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
We could keep these files as templates in the tools directory instead of in each of the project src directories. However, I think it's more intuitive to someone looking at this repo to know that this file will be included in the final nupkg instead of conjuring up a file from an arbitrary location.

Reply:
I'm fine either way